specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2025-03-02-preview/managedClusters.json

```diff
@@ -7657,7 +7657,7 @@
         "values": [
           {
             "value": "DynamicIndividual",
-            "description": "Each pod gets a single IP address assigned. This is better for maximizing a small to medium subnet of size /16 or smaller. The Azure CNI cluster with dynamic IP allocation defaults to this mode if the customer does not explicitly specify a podIPAllocationMode"
+            "description": "Each node gets allocated with a non-contiguous list of IP addresses assignable to pods. This is better for maximizing a small to medium subnet of size /16 or smaller. The Azure CNI cluster with dynamic IP allocation defaults to this mode if the customer does not explicitly specify a podIPAllocationMode"
```

Comment:
please update swagger file under folder 2025-04-02-preview in dev branch [FumingZhang-containerservice-Microsoft.ContainerService-2025-04-02-preview](https://github.com/Azure/azure-rest-api-specs/tree/FumingZhang-containerservice-Microsoft.ContainerService-2025-04-02-preview)